shared/translations/languages/pt.txt

```diff
@@ -1256,7 +1255,6 @@
     "Available to sell": "Disponível para vender",
     "Fee": "Taxa",
     "Each transaction will be subject to a small amount of fees.": "Jede Transaktion unterliegt einer geringen Gebühr.",
-    "Transfer to your bank account": "",
```

Comment:
Why is pt a txt file and why are those removed? cleanup?

Reply:
temporarily changed to disable